enhancements/installer/packet-ipi.md

```diff
@@ -128,20 +179,36 @@ may make undertake substantial design changes.  This project will either need to
 adopt and maintain the current releases, or adopt newer releases developed
 alongside this project.
 
-The CI will need to assure that Packet resources are cleanly released after failed and successful builds.  Packet is currently used in the baremetal provisioning tests, so there is some prior art available.  If the same account is used for testing the IPI, it may need account quota increases.
+The CI will need to assure that Packet resources are cleanly released after
+failed and successful builds.  Packet is currently used in the bare metal
+provisioning tests, so there is some prior art available.  If the same account
+is used for testing the IPI, it may need account quota increases.
 
 ## Design Details
 
+### Open Questions
+
+- Is the Packet CCM necessary?
+- Is the Packet CSI necessary? If not, how will the Openshift image registry be
+  created.
+- Can the sig-cluster-lifecycle ClusterAPI and OpenShift MachineAPI converge to
+  prevent maintaining two similar but different ClusterAPI implementations?
+- How will the RHCOS image be served?
+- How will Ignition be started and configure nodes appropriately? (restating - DHCP is not available but a metadata service is)
```

Comment:
There's no bootstrapping problem here.  The system comes up, DHCPs for its public IPv4 address, Afterburn configures other addresses by querying the metadata service, then Ignition runs.